templates/2015/sponsoring.html.ep

```diff
@@ -0,0 +1,60 @@
+% layout 'default';
+% title 'Sponsoring Mojoconf';
+
+%= include 'partial/header';
+%= include 'partial/top_image';
+
+<div class="content-wrapper">
+  %= include 'partial/mojoconf-content-header', header => 'Sponsoring Mojoconf'
+  <div class="content">
+    <section>
+
+      <h3><i class="fa fa-map-marker"></i>About Mojoconf</h3>
+      <p>
+        Mojoconf is an annual conference that brings together WebDev and WebOps
+        professionals from around the globe. It focuses on the Mojolicious web
+        development framework, which enables developers and designers to move
+        quickly from proof-of-concept to production quality web applications.
+      </p>
+
+      <h3><i class="fa fa-map-marker"></i>Get the Prospectus</h3>
+      <p>
+        The <%= link_to '/2015/prospectus.pdf' => ( target => '_blank' ) => begin %>
```

Comment:
Do you need target blank? Is that some magical feature where the pdf is loaded inside the browser maybe?